src/OpenTelemetry.Resources.Container/K8sMetadataFetcher.cs

```diff
@@ -0,0 +1,76 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.IO;
+using System.Text;
+
+namespace OpenTelemetry.Resources.Container;
+
+internal sealed class K8sMetadataFetcher : IK8sMetadataFetcher
+{
+    private const string KubernetesServiceHostKey = "KUBERNETES_SERVICE_HOST";
+    private const string KubernetesServicePortKey = "KUBERNETES_SERVICE_PORT_HTTPS";
+    private const string KubernetesHostnameKey = "HOSTNAME";
+    private const string KubernetesPodNameKey = "KUBERNETES_POD_NAME";
+    private const string KubernetesContainerNameKey = "KUBERNETES_CONTAINER_NAME";
```

Comment:
Consider: KubernetesServiceHostKey -> KubernetesServiceHostEnvironmentVariable/ KubernetesServiceHostEnvVar

Same for others.

Reply:
Renamed.